LICENSE.md

```diff
@@ -0,0 +1,22 @@
+# The MIT License (MIT)
+
+Copyright (c) 2019 Fang19
```

Comment:
Hello there, it seems a typo here (`Fang19` -> `Feng19`). But we prefer to use the organization name as `eDragonConnect`.